src/omdbapi.js

```diff
@@ -0,0 +1,20 @@
+const API_KEY = "6de06ef5"
+const API_BASE = "https://www.omdbapi.com/?apikey=" + API_KEY;
+
+export async function loadIMDBMovieRating (id, query = "&i=") {
+
+    const res = await fetch(API_BASE + query + id);
+    const data = await res.json();
+
+    if (data.Response == "False") {
+
+        return data;
+
+    } else {
+        // Because IMDB rating is 0-10 and we want a 0-5 rating, to be consistent with our own rating 
+        return data.imdbRating / 2;
+    }
```

Comment:
Returning two very different types of return values is risky, because it requires the users of this function to be aware of all of the different types. If there is an error, it should be indicated by something falsy, e.g. `false`, `null` or `-1`, so that users of the function can easily handle the error.